apps/comments-ui/src/AppContext.ts

```diff
@@ -82,6 +83,7 @@ export type EditableAppContext = {
     labs: LabsContextType,
     order: string,
     adminApi: AdminApi | null,
+    replyToHighlight: string | null,
```

Comment:
Can we generalise to `commentToHighlight`? I don't think there's anything specific about replies in the code  other than _right-now_ that's our only use-case, seems like the code could be simplified and made more re-usable in future by allowing any comment to be highlighted.